src/lib/meter/meter/_core.scss

```diff
@@ -0,0 +1,211 @@
+@use '../../core/styles/spacing';
+@use '../../core/styles/typography';
+@use './token-utils' as *;
+
+@forward './token-utils';
+
+@mixin host {
+  display: inline;
+}
+
+@mixin vertical-host {
+  display: block;
+}
+
+@mixin base {
+  box-sizing: border-box;
+}
+
+@mixin vertical-base {
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: end;
+  width: fit-content;
+  height: 100%;
+  max-width: 100%;
+}
+
+@mixin heading {
+  @include typography.style(label1);
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  line-height: normal;
+}
+
+@mixin vertical-heading {
+  flex-direction: column;
+  align-items: start;
+}
+
+@mixin heading-with-content {
+  margin-bottom: #{spacing.variable(xxsmall)};
+}
+
+@mixin vertical-heading-with-content {
+  margin-bottom: 0;
+  margin-left: #{spacing.variable(xsmall)};
+}
+
+@mixin label {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+@mixin value {
+  color: var(--forge-theme-text-medium);
+}
+
+@mixin track {
+  position: relative;
+  block-size: #{token(height)};
+  border-radius: #{token(shape)};
+  overflow: hidden;
+  background: #{token(background)};
+}
+
+@mixin vertical-track {
+  width: #{token(height)};
+  height: 100%;
+}
+
+@mixin bar {
+  position: absolute;
+  left: -100%;
+  width: 100%;
+  height: 100%;
+  border-radius: #{token(inner-shape)};
+  box-shadow: #{token(inner-elevation)};
+  background: #{token(color)};
+  translate: var(--_meter-percentage) 0;
+}
+
+@mixin vertical-bar {
+  left: initial;
+  top: 100%;
+  translate: 0 calc(-1 * var(--_meter-percentage));
+}
+
+@mixin theme($theme, $muted: false) {
+  @include override(background, var(--forge-theme-#{$theme}-container-#{if($muted, 'minimum', 'low')}), value);
```

Comment:
I'm not sure we have a good pattern for this, so maybe that's why you did it this way, but we typically avoid writing `var()` references directly in favor of a mixin which generates the token. So we could use a mixin to handle generating the full `var()` expression including the fallback value. 

Same for the other usages in this file.